src/renderer/dataManager.ts

```diff
@@ -116,7 +120,7 @@ export class DataManager<T extends YatsugiGroup | YatsugiTool> {
         await this.update();
     }
 
-    private async validLentItem(toolIDs: string[]): Promise<boolean> {
+    async validLentItem(toolIDs: string[]): Promise<boolean> {
```

Comment:
It seems that the token, `private` you wanted to remove is not this one.
And also it makes this build result failed.